### ISSUE TITLE
Problem with the test vectors.

### ISSUE THREAD

**Issue by mbaillif** [reds-heig/logisim-evolution#193]
Problem with the test vectors.
In some cases, the test does not work properly.
Example at the top, the circuit with its vector file and the correct result.

At the bottom the introduction of three "not" doors will generate errors.
It can be seen that in the "Ob" column these errors are not detected.
All columns should be red.
Apparently when there is an error in the first column the errors of the second column are not detected.

![image](https://user-images.githubusercontent.com/42096681/49178484-29432680-f350-11e8-9767-9efc05e47a10.png)

**Comment by maehne** [comment]
Thanks for the report! Could you please provide the corresponding Logisim circuit file(s) to facilitate reproducing the error?

**Comment by mbaillif** [comment]
originally I observed the problem on a more complex circuit.
I have reduced this circuit to its simplest expression to test this problem.
There remain only three inputs connected to three outputs with the test vectors file as shown in the figure.
I added "Not" gates so that the outputs are in error with respect to the vector file.
I could also edit the vector file without modifying the schema.
This problem arises as well for 1/0 errors as for E or X
